clusterloader2/cmd/clusterloader.go

```diff
@@ -61,6 +61,7 @@ func initClusterFlags() {
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "KUBECONFIG", "", "Path to the kubeconfig file")
 	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", "NUM_NODES", 0, "number of nodes")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "PROVIDER", "", "Cluster provider")
+	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.EtcdInsecurePort, "etcdinsecureport", "ETCD_INSECURE_PORT", "2381", "Inscure http port")
```

Comment:
nit: make it `etcd-insecure-port` so it reads better

Reply:
done